test/e2e/auth/service_accounts.go

```diff
@@ -870,6 +857,50 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 		framework.ExpectEqual(eventFound, true, "failed to find %v event", watch.Deleted)
 	})
+
+	ginkgo.It("should guarantee kube-root-ca.crt exist in any namespace", func() {
+		const rootCAConfigMapName = "kube-root-ca.crt"
+
+		_, err := f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Get(context.TODO(), rootCAConfigMapName, metav1.GetOptions{})
```

Comment:
the configmap might not exist instantly in the new test namespace (it's created by a controller), so this should be a wait.PollImmediate with a wait.ForeverTestTimeout

```go
framework.ExpectNoError(wait.PollImmediate(time.Second, wait.ForeverTestTimetout, func() (bool, error) {
  _, err := f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Get(context.TODO(), rootCAConfigMapName, metav1.GetOptions{})
  if err == nil {
    return true, nil
  }
  if errors.IsNotFound(err) {
    framework.Logf("root ca configmap not found, retrying")
    return false, nil
  }
  return false, err
}))
```

Reply:
done